quarkus-file-vault.tf

```diff
@@ -0,0 +1,39 @@
+# Create repository
+resource "github_repository" "quarkus_file_vault" {
+  name                   = "quarkus-file-vault"
+  description            = "A cool description"
```

Comment:
Go ahead and add a description here

Reply:
I propose `Credentials Provider which extracts secrets from Java KeyStore` for a start.

Reply:
`A Vault CredentialsProvider implementation which extracts secrets from Java KeyStore` ?